courses/machine_learning/deepdive2/time_series_prediction/1_data_exploration.ipynb

```diff
@@ -0,0 +1,1354 @@
+{
```

Comment:
<p>Chris, this notebook all looks good to me. Since I lasted looked it over, it's been decided that we need to number the learning objectives and then indicate which code blocks correspond to which objectives. </p><p> </p><p>It doesn't need to be a 1-1 mapping; i.e. Learning Objective 1 could correspond to TODO 1a, TODO 1b, etc. </p><p> </p><p>We just need to give an indication of where the code for each Learning Objective would relate to so the lab development team will know where to put TODOs for students. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/646/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>